scripts/change-fonts-cjk.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+set -e
+
+FONTDIR="./fonts"
+
+mkdir -p "${FONTDIR}"
+
+# download filename url
+download() {
+  ## Download if newer, and if curl fails, clean up and exit
+  curl --fail --compressed -A "get-fonts.sh/osm-carto" -o "$1" -z "$1" -L "$2" || { echo "Failed to download $1 $2"; rm -f "$1"; exit 1; }
+}
+
+case "$1" in
+  ja)
+    download "${FONTDIR}/NotoSansCJKjp-Regular.otf" "https://github.com/googlefonts/noto-cjk/raw/main/Sans/OTF/Japanese/NotoSansCJKjp-Regular.otf"
+    download "${FONTDIR}/NotoSansCJKjp-Bold.otf" "https://github.com/googlefonts/noto-cjk/raw/main/Sans/OTF/Japanese/NotoSansCJKjp-Bold.otf"
+    sed -i "s/Noto Sans CJK (JP|KR|SC|TC|HK)/Noto Sans CJK JP/g" ./style/fonts.mss
```

Comment:
Can you turn the first part of the pattern into a variable so it's only defined once.

Reply:
I added backslashes before all of parentheses and pipes so the patterns are correctly recognized. I tested it on Ubuntu 18.04, and it works well.